source/exercises/mailroom.rst

```diff
@@ -362,3 +362,51 @@ I like to say: "If it's hard to test, it's not well structured"
 
 Put in the tests **before** you make the other changes below - that's much of the point of tests -- you can know that you haven't broken anything when you refactor!
 
+Guidelines
+-----------
+
+Here are some suggestions on what should be refactored in your mailroom code.
+
+At this point, you should avoid unit testing code with ``input`` or ``print`` functions, these pieces require more advanced unit testing methodologies which will be revisited in future courses.
```

Comment:
I think we need to be explicit about the fact that some of the refactoring can be guided by making surethat there functions with input and print have little logic in them as possible. -- so rather than simply skipping funcitons that have a print, they need to refactor the print out of those functions.

Reply:
I think I see what you're saying and how my wording can give the wrong impression. Going to work on rewording this.